### ISSUE TITLE
Failed upgrade to Craft Commerce 3

### ISSUE THREAD

**Issue by mattmurtaugh** [FosterCommerce/shipstation-connect#15]
Are there plans to update this plugin for commerce 3? I tried install it on a development site I'm working on and get the following error:

``` shell
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Installation request for fostercommerce/shipstationconnect ^1.2 -> satisfiable by fostercommerce/shipstationconnect[1.2.0, 1.2.1, 1.2.2, 1.2.3, 1.2.4, 1.2.5, 1.2.6, 1.2.7].
    - Can only install one of: craftcms/commerce[v2.x-dev, 3.0.5].
    - Can only install one of: craftcms/commerce[v2.x-dev, 3.0.5].
    - Can only install one of: craftcms/commerce[v2.x-dev, 3.0.5].
    - Conclusion: install craftcms/commerce v2.x-dev
    - Installation request for craftcms/commerce (locked at 3.0.5, required as ^3.0.0) -> satisfiable by craftcms/commerce[3.0.5].
```

**Comment by NixonGit** [comment]
Might there be an update coming soon? Experiencing a similar installation problem as above. Thanks

**Comment by sjcallender** [comment]
We'll be upgrading the plugin to be Commerce 3 compatible in March.
Thanks for using the plugin and reaching out.

**Comment by johnnynotsolucky** [comment]
Resolved in 1fd02341d5419196a43f4773c31a510d6a252280
